src/components/contextMenu/AposContextMenuTip.vue

```diff
@@ -1,20 +1,32 @@
 <template>
-  <svg :class="[alignmentModifier, originModifier]" class="apos-context-menu__tip" width="27px" height="13px" viewBox="0 0 27 13" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+  <!-- Disabling since the SVG is mostly not active vue template code. -->
```

Comment:
Should we add an ignore pattern for this so we don't have to manually add this in the future? Or should `svg`s live somewhere else and be imported?

Reply:
The trouble is that some of these SVGs do have reactive attributes, so I don't think we can store them elsewhere and import. There might be some regex way to do that. I'm not sure yet that it's worth the effort.

Reply:
I'd be happy to be wrong there. I just don't want to fight with regex for this right now.

Reply:
Should be a way to do that without a regex. I guess we can look at that later.